app/src/main/java/com/example/oddhours/ui/addjob/AddJobFragment.kt

```diff
@@ -106,8 +117,10 @@ class AddJobFragment : Fragment() {
 //                    clearFields()
                 }
             } else if (companyName == "") {
+                hideKeyboard()
                 Toast.makeText(activity, "Please enter a name for the company", Toast.LENGTH_LONG).show()
             } else if (location == "") {
+                hideKeyboard()
```

Comment:
i don't think you want the keyboard to disappear every time we notify the user about an error they made. if they want to correct it they'd have to tap again to bring up the keyboard.. i think we can remove all the `hideKeyboard()` calls from the validation section

Reply:
yaa makes sense. I was testing for the Toast msg which isnt working for me